pkg/generate/command.go

```diff
@@ -0,0 +1,35 @@
+package generate
+
+import (
+	"fmt"
+	"log"
+	"github.com/budimanjojo/talhelper/pkg/config"
+	"github.com/budimanjojo/talhelper/pkg/talos"
+)
+
+func GenerateCommand(c *config.TalhelperConfig, outDir string, generateForNode string, generateApply bool, generateUpgrade bool) error {
+	if !generateApply && !generateUpgrade {
+		fmt.Printf("Must select one of `--apply` of `--upgrade`\n")
```

Comment:
Like this, you can just `return fmt.Errorf("Must select one of --apply or --upgrade")`